quickstarts/microsoft.databricks/databricks-workspace-with-dbfs/azuredeploy.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
```

Comment:
Since this template was authored with bicep, can you please include the original bicep file and remove the JSON file? We will generate the JSON as part of our CI process when this code gets checked in.